<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCadastro</class>
 <widget class="QMainWindow" name="TelaCadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="CriarSobreNM">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sbNome">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>Sobrenome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="criar_cpf">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Criar_Nome">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nome">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpf">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>691</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_senha">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>Confirmar CPF:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="criar_senha">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_usuario">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
      <property name="text">
       <string>Criar Usu√°rio:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="criar_usuario">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 65, 172);</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_voltar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>560</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(145, 65, 172);</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cadastrar">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>560</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 65, 172);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>frame</zorder>
    <zorder>btn_voltar</zorder>
    <zorder>btn_cadastrar</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
